src/containers/ClientDeviceDetails/index.js

```diff
@@ -52,8 +52,18 @@ const ClientDeviceDetails = ({
     totalTxPackets,
   } = details?.metricDetails || {};
 
+  const getStatus = () => {
```

Comment:
you're calling the same function twice, with the same result; twice as much computation on every render. Use a `useMemo`